### PR TITLE
Replaced suzanne monkey head with actual tank model.

### PR DESCRIPTION
**Pull request by LarsDu**
Felt bad for this project since it looks like the monkey head is still being used as a placeholder, so here is a hovertank mesh.

Also, the loading code looks a bit overly complex and hard to parse. I'd recommend simplifying things a bit.

### CODE REVIEW

**Review by Indy2222 — CHANGES_REQUESTED** [review]
Thank you for the contribution ❤️!

You are right, we are still at the very beginning regarding art and overall visual design. This has to change since it might put people off.

Regarding the simplification, the first step is to document some not-yet-documented parts, e.g. #276. If you have other suggestions, feel free to open a discussion or submit another PR.

The 3D models themselves are already documented here [CONTRIBUTING.md#3d-models](https://github.com/DigitalExtinction/Game/blob/main/CONTRIBUTING.md#3d-models). Though, there is still this small issue which I hope to fix soon #262.

Suggestions:

* [ ] Would it be possible to include `Relates to #238` in the commit message (as per [CONTRIBUTING.md#pull-requests--git](https://github.com/DigitalExtinction/Game/blob/main/CONTRIBUTING.md#pull-requests--git))?
* [ ] Please track the .blend file with LFS (it is almost 1MiB large).
* [ ] Make sure that the bottom of the tank is on XZ plane ([CONTRIBUTING.md#3d-models](https://github.com/DigitalExtinction/Game/blob/main/CONTRIBUTING.md#3d-models)).
* [ ] The muzzle coordinates seems to be off (inside `assets/objects/monk_tank.obj.json`). It should be placed at the tip of the cannon, i.e. arrow 1) in the picture.
* [ ] The convex hull doesn't fully contain the object. See arrow 2) in the picture pointing to the white circumscribed circle around the convex hull.

Note that the issues might have originated in usage of `de_tools` crate (if you used it) due to this bug #279 (just created). Independently to the issue, I suggest to reset model scale (in Blender) to 1:1 before export to GLB (if that is not already the case).

Let me know how you want to proceed. I take over the PR if you want me to.

tank

**Comment by LarsDu** [comment]
It's better if you take over the PR. I'm just here to make a small contribution. Feel free to cut .blend out of source control entirely
